src/status_im/ui/screens/communities/reorder_categories.cljs

```diff
@@ -59,20 +59,20 @@
   [{:keys [id name community-id]} is-active? drag]
   (let [background-color (if is-active? colors/gray-lighter colors/white)
         category-none?   (string/blank? id)]
-    [:<>
+    [:<> {:style {:flex 1}}
      [quo/separator]
      [rn/view {:accessibility-label :category-item
                :style               (merge styles/category-item
                                            {:background-color background-color})}
-      (if category-none?
+      (if (= @selected-tab :chats)
         [icons/icon :main-icons/channel-category {:color colors/gray}]
         [rn/touchable-opacity
          {:accessibility-label :delete-category-button
-          :on-press            #(show-delete-catgory-confirmation community-id id)}
+          :on-press            #(show-delete-category-confirmation community-id id)}
          [icons/icon :main-icons/delete-circle {:no-color true}]])
       [rn/view {:flex 1}
        [rn/text {:style {:font-size 17 :margin-left 10 :color colors/black}} name]]
-      (when (and (not category-none?) @collapse-chats?)
```

Comment:
don't remove this, otherwise it will also show drag handles on `edit chats` screen

Reply:
I only removed @collapse-chats? as its not needed anymore but category-none condition is still present

Reply:
Maybe we can replace this with something like?
` (when (and (not category-none?)  (= @selected-tab :categories))`
Also we are using this check at several places, can we create a function like

``` 
(defn categories-tab? []
(= @selected-tab :categories))
```

Reply:
I think drag handles shouldnt be visible on edit chats screen. I get it I will implement the check